index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
> why not also make skip dynamic?

Can you elaborate on what you mean with this? `skip` accepts a promise as well, so in that sense it is dynamic. But maybe you're hinting at something else :).

> Could you open an issue about this on Listr, so we could continue this discussion there?

Jup, I will. Just want to clarify the `skip` part first :).